src/backend/updating-llvm.md

```diff
@@ -66,9 +66,9 @@ Example PRs look like:
 
 ## Feature updates
 
-> Note that this is all information as applies to the current day in age. This
-> process for updating LLVM changes with practically all LLVM updates, so this
-> may be out of date!
+> Note that this information is as of the time of this writing (December 2018
+> <!-- date: 2018-12 -->. The process for updating LLVM changes with practically
```

Comment:
The date-checker hasn't landed yet, but I think it will soon and this doesn't cause any harm, so I thought it was worth adding a date-check comment.

Reply:
It has landed!